src/content/docs/en/release/diff-fedora.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Differences with Fedora
+description: "Known differences and changes compared with Fedora"
+---
+
+import Alert from "../../../../components/Docs/Alert.astro";
+
+This page documents Ultramarine Linux's significant changes compared with a typical Fedora workstation setup.
+
+Changes listed below are nonexhaustive and cumulative, meaning each section contains changes from all the section below it.
+
+### Emoji meanings
+
+- 📦 This is packaged in [Terra]
+- 🛳️ This ships existing Fedora packages
+- 🔥 This is a significant breaking change
+
+## Ultramarine 40
+
+- 🔥 [Change default package manager to `dnf5`](/en/usage/um40-dnf5) (`dnf5-default-package-manager`)
+- Remove Fedora bookmarks in Firefox (`ultramarine-phony-bookmarks`)
+- Drop Pantheon Edition
+- `ultramarine-fun`
+- 🛳️ `git`
+- 🛳️ XFCE Edition
+- Improve key bindings for `zsh`
+
+## Before Ultramarine 40
+
+- Out of box repositories: [Terra], full set of [RPM Fusion], [Flathub]
+- IP MTU probing enabled (which fixes ubiconnect games)
```

Comment:
Might wanna change to "Ubisoft Connect" to avoid confusion, contractions are good but it confuses normies